cmd/mockery/mockery.go

```diff
@@ -217,3 +218,13 @@ func underscoreCaseName(caseName string) string {
 	rxp2 := regexp.MustCompile("([a-z0-9])([A-Z])")
 	return strings.ToLower(rxp2.ReplaceAllString(s1, "${1}_${2}"))
 }
+
+func filename(name string) string {
+	name = strings.ToLower(name)
```

Comment:
You don't need this anymore, caseName, the argument passed in, is already formatted properly.